command_line/image_viewer.py

```diff
@@ -83,8 +83,14 @@
   .type = int
 gain = 1
   .type = float(value_min=0)
-  .help = "Set gain for the thresholding algorithm. This does not override the"
+  .help = "Set gain for the dispersion algorithm. This does not override the"
           "detector's panel gain, but acts as a multiplier for it."
+n_sigma = 8
+  .type = float(value_min=0)
+blur = narrow wide
+  .type = choice
+n_bins = 100
+  .type = int
```

Comment:
Could these be shared with the definitions in the algorithm itself? So that we can benefit from the help strings here, as well as ensuring that both sets of parameters remain in sync.